apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -1106,6 +1120,26 @@ export class WorkshopForm extends React.Component {
             readOnly={this.props.readOnly}
           />
           <br />
+          {this.state.course === COURSE_BUILD_YOUR_OWN && (
+            <Row>
+              <Col sm={4}>
+                <FormGroup validationState={validation.style.name}>
+                  <ControlLabel>Workshop Name</ControlLabel>
+                  <FormControl
+                    type="text"
+                    value={this.state.name || ''}
+                    id="name"
+                    name="name"
+                    onChange={this.handleFieldChange}
+                    maxLength={255}
```

Comment:
Is this just the default value for this prop? Would love to break this out so it’s not a magic number but that might be too big of a change 😬

Reply:
I saw that the [other text field components](https://github.com/code-dot-org/code-dot-org/pull/62841/files#diff-f0655af74c25a070d365fc92a5d25247d6f3bc0d24e209e3ea0cafdf9fe3f25cR1153) in this file had this same prop (I think to match the 255 character limit of the database entry). I could make it a file constant to clean it up a bit?